src/Traits/ReviewPermission.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\ContentReview\Traits;
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+trait ReviewPermission
+{
+    /**
+     * Whether or not a user is allowed use content review
+     */
+    protected function canUseReviewContent(DataObject $record, ?Member $user = null): bool
+    {
+        // Whether or not the user is a reviewer. User must be allowed to view the page
+        $isReviewer = $record->canView($user) &&
```

Comment:
```suggestion
        $canBeReviewedByMember = $record->canView($user) &&
```
Note - depends on answer to question below

Reply:
removed this as we can just use a ternary on the return.